website/blog/2023-06-11-reborn.md

```diff
@@ -0,0 +1,62 @@
+---
+title: ReasonReact is back (and powered by Melange)
+---
+
+ReasonReact moved to `@rescript/react` after the ReScript announcement. The future of ReasonReact has been uncertain ever since. In the meantime, the Melange team announced the first major release of [Melange](https://anmonteiro.substack.com/p/melange-10-is-here) and took the first steps towards deeper integration with the OCaml ecosystem.
+
+Today, we are happy to announce that ReasonReact is back and will be maintained by the Melange team going forward.
+
+## What's changing?
+
+We released ReasonReact to the [opam repository](https://ocaml.org/p/reason-react/latest). In the Melange docs, we document [how to depend](https://melange.re/v1.0.0/package-management/) on packages in opam.
```

Comment:
There's a newer section with more details in https://melange.re/unstable/package-management/#finding-and-using-melange-compatible-packages, maybe worth linking to it directly.

Reply:
I prefer to keep 1.0.0 stuff under this blog post